.appveyor.yml

```diff
@@ -0,0 +1,27 @@
+# Builds are whole numbered
+version: '{build}'
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
You can do whatever you feel is best, but I think it would be better to keep it as the default, since that's what users will most likely have (and for testing it's normally a good idea to mirror end-user's environments).  But like I said, up to you.  The default is also what AGI developers are also encouraged to keep on their own system.